invoices/invoice.css

```diff
@@ -54,7 +53,7 @@ body {
 
 .client .details, .summary .details {
   padding: 4px;
-  background: #fff;
+  background-color: var(--grist-theme-page-panels-main-panel-bg);
```

Comment:
I think we need to add fallbacks for these. (The variables don't always exist, like when custom CSS is enabled.)

Reply:
Thanks. Done.